src/main/java/org/broadinstitute/hellbender/engine/filters/MetricsReadFilter.java

```diff
@@ -4,6 +4,15 @@
 import org.broadinstitute.hellbender.utils.help.HelpConstants;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
+/**
+ * Filter out reads that:
+ *
+ * <ul>
+ *     <li>Fail platform/vendor quality checks</li>
```

Comment:
After going through the entire list of filters, especially digging about unmapped reads, it occurs to me that SAM flags are not necessarily the only way to define these categories of reads.